scenarioo-client/app/components/detailarea/detail-accordion/detail-accordion.component.html

```diff
@@ -16,15 +16,16 @@
 -->
 
 <accordion-group [attr.aria-expanded]="!isAccordionCollapsed"
-                 [isOpen]=isFirstOpen
-                 panelClass="customClass">
-    <div accordion-heading (click)="isAccordionCollapsed = !isAccordionCollapsed">
-                        <span *ngIf="isAccordionCollapsed">
-                            <i class="fas fa-angle-down"></i>
-                        </span>
+                 [isOpen]="!isAccordionCollapsed"
+                 panelClass="customClass"
+                 [isDisabled]="true">
```

Comment:
why is it always disabled?

Reply:
The idea was to control the collapse/expand and the change of the arrows with the click event. 

As I understood, the accordion collapses and expands by default without a click event, just by clicking on the accordion-heading. To prevent this, the accordion group is always disabled.